lib/index.js

```diff
@@ -59,15 +60,19 @@ class Oppsy extends Events.EventEmitter {
                 this.emit('error', err);
             }
             finally {
-                this._networkMonitor.reset();
+                if(this.config.dontResetNetworkCounters == true){
+                    this._networkMonitor.reset();
+                }
             }
         }, interval);
     }
 
     stop() {
 
         clearInterval(this._interval);
-        this._networkMonitor.reset();
+        if(this.config.dontResetNetworkCounters == true){
```

Comment:
Do we need to do this? After `stop` is called, will the `this.emit` happen one more time? If we don't need it, then we can skip attaching `config` to `this`.